src/rules/selector-descendant-combinator-no-non-space/index.js

```diff
@@ -0,0 +1,46 @@
+import {
+  isStandardSyntaxRule,
+  isStandardSyntaxSelector,
+  parseSelector,
+  report,
+  ruleMessages,
+  validateOptions,
+} from "../../utils"
+import { nonSpaceCombinators } from "../../reference/punctuationSets"
+
+export const ruleName = "selector-descendant-combinator-no-non-space"
+
+export const messages = ruleMessages(ruleName, {
+  rejected: nonSpaceCharacter => `Unexpected non-space "${nonSpaceCharacter}" characters in descendant combinator`,
```

Comment:
We've removed redundancy between the message and the rule name elsewhere, as such I think we could even shorten this to `Unexpected "${nonSpaceCharacter}"`

e.g.

`Unexpected "\t" (selector-descendant-combinator-no-non-space)`